### DISCUSSION TITLE
How to redownload a previously downloaded file?

### DISCUSSION THREAD

**Discussion by ww2283** [sabnzbd/sabnzbd#2669] [General]
I had a movie that I updated with radarr because the quality was supposed to be superior. However, upon close inspection, I determined that it is worse than a previous version from another group. I decided to redownload the previous one. Here my problem occurs: sabnzbd always rejects this download request. Steps I have tried so far:
1. disable dup detection in sab
2. delete that movie entry in radarr and re-add it
3. manually put in sab download in sab interface
I'm asking for help on how to redownload the version I desired that has been downloaded before?

**Comment by DarrenPIngram** [comment]
On 2023-08-30 20:50, ww2283 wrote:
> I'm asking for help on how to redownload the version I desired that
> has been downloaded before?
Had you cleared out your SAB download history? Maybe it is detecting it 
there otherwise?

**Comment by thezoggy** [comment]
> I had a movie that I updated with radarr because the quality was supposed to be superior. However, upon close inspection, I determined that it is worse than a previous version from another group. I decided to redownload the previous one. Here my problem occurs: sabnzbd always rejects this download request. Steps I have tried so far:
> 1. disable dup detection in sab
> 2. delete that movie entry in radarr and re-add it
> 3. manually put in sab download in sab interface
> I'm asking for help on how to redownload the version I desired that has been downloaded before?

You shouldn't use sab dupe detection if your using radarr as it has its own.... 

But anyways how sab dupe detection works is two places. 

One is sabs history, so delete previous download from history of existing.

The other is the nzb backup folder if you have that set, which we compare md5 sum of said nzb to know of it's a match. So you would remove it from there as well. (Config>folders>nzb backup) to know where to look.

**Comment by ww2283** [comment]
> One is sabs history, so delete previous download from history of existing.

Thanks for the both feedbacks, I'm replying once just not to make the thread looking too busy: so 'delete previous history' does that mean I set 0 to Config -> Switches -> Post-Processing -> History Retention? Then I wait after next nzb grab to let the new rule of History Retention be effective?

I will turn off the detect dup thanks for pointing out the radarr's monitoring.

I had tried clear the backup folder but that didn't work on its own, I guess it's the history setting playing its role.

**Reply by thezoggy** [reply]
> > One is sabs history, so delete previous download from history of existing.
> 
> Thanks for the both feedbacks, I'm replying once just not to make the thread looking too busy: so 'delete previous history' does that mean I set 0 to Config -> Switches -> Post-Processing -> History Retention? Then I wait after next nzb grab to let the new rule of History Retention be effective?
> 
> I will turn off the detect dup thanks for pointing out the radarr's monitoring.
> 
> I had tried clear the backup folder but that didn't work on its own, I guess it's the history setting playing its role.

You need to keep sab history for apps like radarr to work as it relies on said history. Now you can just set in radarr to remove the job from sab once it imports the job (or if it failed). Which is usually recommended that way radarr only sees things in sab history that it hasn't processed.
